scripts/validate-eks-ci-role-policy/main.go

```
@@ -0,0 +1,377 @@
+		version.KustomizeVersion != expectedKustomizeVersion {
+		return fmt.Errorf(
+			"unapproved renderer: kubectl=%s kustomize=%s",
+			version.ClientVersion.GitVersion,
+			version.KustomizeVersion,
+		)
+	}
+	return nil
+}
+
+func commandOutput(name string, args ...string) ([]byte, error) {
+	command := exec.Command(name, args...) //nolint:gosec // Fixed binary and repository-controlled arguments.
```

golangci-lint (2.12.2)

[error] os/exec.Command must not be called. use os/exec.CommandContext

(noctx)